docs/integrations/sources/greenhouse.md

```diff
@@ -71,6 +71,7 @@ The Greenhouse connector should not run into Greenhouse API limitations under no
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                              |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0.5.4   | 2024-05-15 | [38258](https://github.com/airbytehq/airbyte/pull/38258) | Update Application incremental stream cursor and add `custom_field` and `keyed_custom_field` columns                                                                                                                                             |
```

Comment:
I saw the other versions were bumped to 1.0 but I still had this one at 0.5.4. Not sure if I should bump this to 1.0 as well.